exhale/graph.py

```diff
@@ -246,6 +248,12 @@ def __lt__(self, other):
         else:
             return self.kind < other.kind
 
+    def __str__(self):
+        return f"{self.kind}: {self.name}"
+
+    def __repr__(self):
+        return self.__str__()
+
```

Comment:
Were you just using these for debugging?  the `__repr__` in particular isn't valid, I'm planning on removing them.

Reply:
Yes, these were just for debugging they could be removed.

Reply:
TODO: remove `__repr__`, make str return more, speciial cases: `self.template` and if function the signature

Reply:
decision reversed, __repr__ kept, improved it in ExhaleNode and use it in testing framework